grpc/py-cubic/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='cobalt-cubic',
+    python_requires='>=3.5.0',
+    version='2.0.0',
+    description='This client library is designed to support the Cobalt API for speech recognition with Cubic',
+    author='Cobalt Speech and Language Inc.',
+    maintainer_email='tech@cobaltspeech.com',
+    url='https://github.com/cobaltspeech/sdk-cubic',
```

Comment:
should this URL be the github URL, or the docs url (https://cobaltspeech.github.io/sdk-cubic) ?

Reply:
I dunno, both the luna-sdk and the whole cubic-sdk-python had the github url

Reply:
Let's use the docs url. The repo url as documentation of the package can be confusing at first glance since the repo is not just the python package